lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/six-month-fraud-check.yaml

```diff
@@ -0,0 +1,141 @@
+# Entry points
```

Comment:
Very minor, but other journeys have another heading for `# Journey states` to distinguish